db/collection_delete.go

```diff
@@ -109,7 +109,7 @@ func (c *collection) deleteWithKey(
 	ctx context.Context,
 	txn datastore.Txn,
 	key core.PrimaryDataStoreKey,
-	status client.DocumentStatus,
+	_ client.DocumentStatus,
```

Comment:
question: Can't we just remove the parameter?

Reply:
done

Reply:
I only flagged this one but there where a few others just in case you didn't fix all of them.